dcnum/feat/feat_moments/mt_legacy.py

```diff
@@ -81,20 +113,9 @@ def moments_based_features(mask, pixel_size):
         if mu_raw['mu02'] > 0:  # defaults to zero
             inert_ratio_raw[ii] = np.sqrt(mu_raw['mu20'] / mu_raw['mu02'])
 
-        # inert_ratio_prnc
```

Comment:
Maybe we should also put the refactoring of the inertia ration computation in the separate PR together with the tilt computation?